cloudslang-all/src/main/java/io/cloudslang/lang/api/SlangImpl.java

```diff
@@ -117,6 +117,12 @@ public void subscribeOnAllEvents(ScoreEventListener eventListener) {
         eventTypes.add(ScoreLangConstants.EVENT_INPUT_END);
         eventTypes.add(ScoreLangConstants.EVENT_OUTPUT_START);
         eventTypes.add(ScoreLangConstants.EVENT_OUTPUT_END);
+        eventTypes.add(ScoreLangConstants.EVENT_BRANCH_START);
+        eventTypes.add(ScoreLangConstants.EVENT_BRANCH_END);
+        eventTypes.add(ScoreLangConstants.EVENT_ASYNC_LOOP_INPUT_START);
```

Comment:
Why do you prefer presenting new event types and not reusing the current input and output types?